api/config.proto

```diff
@@ -13,12 +13,18 @@ message Config {
 
 // SourceRepo contains the required information of the source chart repository
 message SourceRepo {
```

Comment:
As @josvazg mentioned, it might make sense to rename this key name to Source so latter on the struct will be called Source_Repo

Reply:
+1